test/e2e/testsuites/specs.go

```diff
@@ -310,3 +323,65 @@ func CreateVolumeSnapshotClass(client restclientset.Interface, namespace *v1.Nam
 
 	return tvsc, tvsc.Cleanup
 }
+func VerifySuccessfulReplicaAzVolumeAttachments(pod PodDetails, azDiskClient *azDiskClientSet.Clientset, storageClassParameters map[string]string, client clientset.Interface, namespace *v1.Namespace) (bool, *v1alpha1.AzVolumeAttachmentList, error) {
+	if storageClassParameters["maxShares"] == "" {
+		return true, nil, nil
+	}
+
+	var expectedNumberOfReplicas int
+	maxShares, err := strconv.ParseInt(storageClassParameters["maxShares"], 10, 0)
```

Comment:
You can also use `azureutils.GetMaxSharesAndMaxMountReplicaCount`.
This way you can also parse `maxMountReplicaCount` when it is explicitly set in storageclass and is not `maxShares - 1`.